test/verify/check-connection

```diff
@@ -437,23 +437,17 @@ class TestConnection(testlib.MachineCase):
         m.execute("curl -k -c cockpit.jar -s --head --header 'Authorization: Basic {}' https://127.0.0.1:9090/cockpit/login".format(
             base64.b64encode(b"admin:foobar").decode(), ))
         headers = m.execute("curl -k --head -b cockpit.jar -s https://127.0.0.1:9090/")
-        self.assertIn(
-            "default-src 'self' https://127.0.0.1:9090; connect-src 'self' https://127.0.0.1:9090 wss://127.0.0.1:9090", headers)
-        if self.is_pybridge():
-            # We want to make sure we're *not* sending any CORS headers.
-            CORS_HEADERS = [
```

Comment:
Are you sure that you wanted to delete this again in the "rewrite packages.py" commit? You were pouring some effort into it in the "disable CORS" commit. This might just be a rebase error?